### PR TITLE
Task contact

### PR DESCRIPTION
**Pull request by j-add**
Responsive contact section with links to email/social channels.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -273,91 +273,64 @@ header {
...
-}
-
-.mailIcon {
-	font-size: 9rem;
-}
-
-.contactText {
-	font-size: 1.9rem;
+	box-shadow: inset 0px 12px 24px -12px rgba(0, 0, 0, 0.5);
+	-webkit-box-shadow: inset 0px 12px 24px -12px rgba(0, 0, 0, 0.5);
```
Comment (luke-grayland) [inline style.css:293]: should box-shadow go below -webkit-box-shadow?

File: index.html [hunk 2]
```diff
@@ -92,25 +92,16 @@ <h3 class="projectName">Project One</h3></a>
...
-				<div class="contactTitle">
-				</div>
+				<h2 class="contactTitle">Contact me</h2>
+				<p class="contactDesc">If you'd like to get in touch, feel free to drop me an email. If you'd like to keep an eye out for future projects, take a look at my social channels.</p>
 			</div>
-			<div class="socialContainer">
+			<div class="contactBtnContain">
+				<a href="mailto:jordanaddis@gmail.com"><i class="contactBtn far fa-envelope"></i>
+				<a href="https://github.com/j-add" target="_blank"><i class="contactBtn fab fa-github socialIcon"></i></a>
+				<a href="http://linkedin.com" target="_blank"><i class="contactBtn fab fa-linkedin-in socialIcon"></i></a>
```
Comment (luke-grayland) [inline index.html]: is this supposed to point to your profile on linkedin?

File: style.css (hunk 1 above)
Comment (j-add) [reply style.css:293]: Yeah, good spot. I'll fix this.

File: index.html (hunk 2 above)
Comment (j-add) [reply index.html]: It is. I'll add it in and change to a https link.

**Pull request merged**